src/utils/ExtrusionJunction.cpp

```diff
@@ -20,4 +20,17 @@ ExtrusionJunction::ExtrusionJunction(const Point p, const coord_t w, const coord
       region_id(region_id)
 {}
 
+void ExtrusionJunction::Swap(ExtrusionJunction* other) {
+  if (other == this) return;
+  InternalSwap(other);
```

Comment:
code-style naming we use lowerCamelCase for functions:
```suggestion
  internalSwap(other);
```

Reply:
Code was removed, so this is no longer necessary.